src/table/column.rs

```diff
@@ -74,6 +74,15 @@ pub enum PgInterval {
     MinuteToSecond,
 }
 
+#[derive(Debug, Clone)]
+pub enum BlobSize {
+    Default(Option<u32>), // Backward compatibility: MySQL & SQLite support `binary(length)` column type
+    Tiny,
+    Blob,  //default blob, maybe need a better name
+    Medium,
+    Long,
+}
```

Comment:
Hey @hunjixin, all in all it looks good! I have one question. If we have `BlobSize::Blob`, should we make the "default" to `BlobSize::Default(u32)`? Removing `Option<T>`. Basically, `BlobSize::Default(None)` will be `BlobSize::Blob`.

Reply:
Makes sense. Actually seems like `BlobSize::Default` can be removed, we only need one `BlobSize::Blob(Option<u32>)` (like before)

Reply:
got

Reply:
updated, very cool idea.

Reply:
Thanks!